be/tx_group_fom.c

```diff
@@ -291,43 +291,6 @@ static void be_tx_group_fom_handle(struct m0_sm_group *gr,
 	m0_fom_ready(&m->tgf_gen);
 }
 
-/*
- * Wakes up tx_group_fom iff it is waiting.
- * It is possible that multiple fom wakeup asts are posted through different
- * code paths. Thus we avoid waking up of already running FOM.
- */
-static void be_tx_group_fom_iff_waiting_wakeup(struct m0_fom *fom)
-{
-	M0_ENTRY();
-	if (m0_fom_is_waiting(fom)) {
-		M0_LOG(M0_DEBUG, "waking up");
-		m0_fom_ready(fom);
-	}
-	M0_LEAVE();
-}
```

Comment:
Has this been discussed with @max-seagate?

Reply:
I asked him to review, yes.